product_docs/docs/pem/9/pem_rel_notes/950_rel_notes.mdx

```diff
@@ -0,0 +1,26 @@
+| Enhancement  | Added a new alert template to validate the password expiry for the users specific to EDB Postgres Advanced Server. Also modified the alert template `A user expired in N days` to `Number of users expiring in N days` that checks the number of users getting exprired. |
+| Enhancement  | Added a button on the Manage Probes window that resets the probe configurations to default.                                                                                                                                                                              |
+| Enhancement  | Added a `Cleared Alert` button in notifications, to mute the cleared alerts. This enhancement provides the flexibility to avoid getting notifications for the cleared alerts.                                                                                            |
+| Enhancement  | You can now create your own web server certificates as EDB details are no longer added to the web server certificates generated by PEM.                                                                                                                                   |
+| Enhancement  | Added a capability to identify the user who changed the blackout status, date, and time.                                                                                                                                                                                 |
+| Bug&nbsp;fix | Fixed an issue where the deleted custom probe was still visible in probe dependency tab of Alert Templates.                                                                                                                                                              |
+| Bug&nbsp;fix | Fixed the missing permission issue by granting the pem_agent user permission to `custom_email_template` table.                                                                                                                                                           |
+| Bug&nbsp;fix | Excluded 0 sized tables from PEM alert template `largest index by table-size percentage`. Also added the detailed SQL statement for this template.                                                                                                                       |
+| Bug&nbsp;fix | You can now use `v11` as the REST endpoint from the cli.                                                                                                                                                                                                                 |
+| Bug&nbsp;fix | Log in user must have appropriate roles to access `Failover Manager Utilities` and `Database server service management`.                                                                                                                                                 |
```

Comment:
This statement doesn't tell me what we changed. @edbshubham could you clarify and suggest some wording?

Reply:
The above comment pertains to line 22.

Reply:
The user should be a pem_user and should be a member of pem_manage_efm role to access EFM features.
And to start/stop the database service from PEM GUI user should be member of pem_server_service_manager role.